src/util/tile.ts

```diff
@@ -26,79 +35,141 @@ export function showBroadcast(
 }
 
 export function stopBroadcast() {
-  console.log("Stopping broadcast");
   broadcastDiv.style.visibility = "hidden";
   broadcastVideo.srcObject = null;
   broadcastDiv.draggable = false;
 }
 
-export function showZonemate(
+// showCamera shows a camera video tile in the zonemates div
+export function showCamera(
   sessionID: string,
   name: string,
   videoTrack?: MediaStreamTrack,
   audioTrack?: MediaStreamTrack
 ) {
-  let zonemate = <HTMLDivElement>(
-    document.getElementById(getZonemateTagID(sessionID))
-  );
-  if (!zonemate) {
-    zonemate = createZonemate(sessionID, name);
-  }
-
-  const vid = <HTMLVideoElement>(
-    document.getElementById(getVideoTagID(sessionID))
-  );
-
   const tracks: Array<MediaStreamTrack> = [];
   if (videoTrack) tracks.push(videoTrack);
   if (audioTrack) tracks.push(audioTrack);
+  showZonemate(ZonemateTileKind.Camera, sessionID, name, tracks);
+}
+
+// showScreenShare shows a screen share video tile in the zonemates div
+export function showScreenShare(
+  sessionID: string,
+  name: string,
+  videoTrack?: MediaStreamTrack
+) {
+  const tracks: Array<MediaStreamTrack> = [];
+  if (videoTrack) tracks.push(videoTrack);
```

Comment:
I haven't seen us using single-line if-statements before,

Reply:
I _think_ I've seen this in other demos and also in pluot-core itself (but I could be wrong). For me this just looks more readable here as we are doing two nearly identical push operations right next to each other. Is there a reason not to use them?

Reply:
The rule of thumb I've used for if statements is a single line if it's just a return and braces if there's any action. 

```
if (whatever) return;
```
We don't have this sort of thing in the style guide as of yet so it's pretty subjective.